### PR TITLE
docs: update doc image

### PR DESCRIPTION
**Pull request by oil-oil**
Changes: Change the pictures in the article into English https://apisix.apache.org/blog/blog/2021/06/28/why-we-need-apache-apisix/

Screenshots of the change: no

### CODE REVIEW

**Comment by membphis** [comment]
image

**Comment by oil-oil** [comment]
> image

Done.

**Comment by oil-oil** [comment]
@juzhiyuan PTAL.

File: website/blog/2021/06/28/why-we-need-Apache-APISIX.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Hello everyone, I'm happy to share a topic that I'm excited about, "Why do you n
 The reason why we are doing a replacement project for NGINX and Kong is actually related to the background of our backend architecture evolution, and I will start by sharing with you the backend architecture evolution process, which is very important.
 
 I'll start by sharing with you the evolution of the backend architecture, which is very important.
-![membphis](https://static.apiseven.com/202108/1646299637523-b074acde-e902-483b-bc27-7ebe75e675cb.jpg)
+![membphis](https://static.apiseven.com/202108/1646320281711-c7e446ce-91f3-4494-93de-8c4b9f7936a2.jpg)
```
Comment (membphis) [inline website/blog/2021/06/28/why-we-need-Apache-APISIX.md]: image

Comment (oil-oil) [reply website/blog/2021/06/28/why-we-need-Apache-APISIX.md]: > image

It means change to "Apache Apisix PMC"?

Comment (juzhiyuan) [reply website/blog/2021/06/28/why-we-need-Apache-APISIX.md]: It should be `Apache APISIX PMC` or `Apache APISIX PMC Member`

File: website/blog/2021/06/28/why-we-need-Apache-APISIX.md [hunk 2]
```diff
@@ -59,27 +57,27 @@ Let's look at the left side of the more interesting JAVA, Spring Cloud built-in
 
 Finally, the bottom right part of the service grid, for the service grid has formed a choice istio (CP) + envoy (DP). Later we see the Alibaba open source mosn, in a nutshell: Golang version of envoy.
```
Comment (SylviaBABY) [inline website/blog/2021/06/28/why-we-need-Apache-APISIX.md]:
```suggestion
Finally, the bottom right part of the service grid, for the service grid has formed a choice Istio (CP) + Envoy (DP). Later we see the Alibaba open source mosn, in a nutshell: Golang version of envoy.
```

File: website/blog/2021/06/28/why-we-need-Apache-APISIX.md [hunk 3]
```diff
@@ -129,43 +127,43 @@ High performance, dynamics, and an active community are the trump cards of APISI
...
 
 For traditional LB and API Gateway scenarios, APISIX has the advantage of going from static to all dynamic, no more reloads, as many tech companies start with a half hour NGINX reload. The aforementioned canary release scenario of moduloing based on request id can be easily done in APISIX using fine-grained routing.
 
-![API gateway](https://static.apiseven.com/202108/20210625020.png)
+![API gateway](https://static.apiseven.com/202108/1646300936549-4b27533d-1a2e-4d65-8c70-470807493d95.jpg)
 
-![APISIX Ingress Controller](https://static.apiseven.com/202108/20210625021.png)
+![APISIX Ingress Controller](https://static.apiseven.com/202108/1646300936551-b463c6b9-7def-432d-b4a4-0bf26de28b38.jpg)
 
 APISIX Ingress Controller solves all the problems mentioned above, and inherits all the advantages of APISIX, in addition to supporting native k8s CRD for easy migration.
```
Comment (SylviaBABY) [inline website/blog/2021/06/28/why-we-need-Apache-APISIX.md]:
```suggestion
APISIX Ingress Controller solves all the problems mentioned above, and inherits all the advantages of APISIX, in addition to supporting native K8s CRD for easy migration.
```

File: website/blog/2021/06/28/why-we-need-Apache-APISIX.md (hunk 1 above)
Comment (oil-oil) [reply website/blog/2021/06/28/why-we-need-Apache-APISIX.md]: > It should be `Apache APISIX PMC` or `Apache APISIX PMC Member`

get

**Review by yzeng25 — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**